website/docs/r/netapp_volume_quota_rule.html.markdown

```diff
@@ -0,0 +1,75 @@
+---
+subcategory: "NetApp"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_netapp_volume_quota_rule"
+description: |-
+  Manages a Volume Quota Rule.
+---
+
+# azurerm_netapp_volume_quota_rule
+
+Manages a Volume Quota Rule.
+
+## Example Usage
+
+```hcl
+resource "azurerm_netapp_volume_quota_rule" "example" {
```

Comment:
And here we want to list all the resources leading up to this and not use `example` as an attribute value